package.json

```diff
@@ -8,12 +8,12 @@
     "test": "eslint ."
   },
   "devDependencies": {
-    "eslint": "^2.0.0",
-    "eslint-plugin-react": "^4.0.0"
+    "eslint": "^3.1.1",
+    "eslint-plugin-react": "^6.0.0-alpha.1"
```

Comment:
But yes, I expect it to be stable before we do v3, especially if you want to also make som rule changes and not just dep updates 😄